src/index.html

```diff
@@ -1,5 +1,8 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
```

Comment:
The opening `<html>` tag should be followed by a 2-space indentation for its attributes to maintain proper indentation as per the HTML Formatting checklist.